ws-nextjs-app/pages/[service]/new_live/[id]/LivePageLayout.tsx

```diff
@@ -54,6 +80,8 @@ const LivePage = ({
         <Heading level={1}>{title}</Heading>
         {/* Text as="p" used as placeholder. Awaiting screen reader UX and UX */}
         <Text as="p">{description}</Text>
+        {/* @ts-expect-error Optimo type nested hell, text doesn't exist on model */}
```

Comment:
Is there a plan to resolve this properly at somepoint?

Reply:
This shouldn't be an issue now, the issue lies with the BFF API and weird optimo types. And then the discussion of unifying types between the two will come into play

Reply:
Scratch that the API has been simplified so its not an issue on the BFF anymore